docs/partials/link.hbs

```diff
@@ -0,0 +1,25 @@
+{{! usage: link to="namepath" html=true/false caption="optional caption"~}}
+
+{{~#if html~}}
+<code>
+
+{{~#link to~}}
+{{#if url~}}
+<a href="http://www.baidu.com/{{{url}}}">{{#if ../../caption}}{{../../../caption}}{{else}}{{name}}{{/if}}</a>
```

Comment:
baidu? WTF?

Reply:
o, it's just a test file... I remembered I deleted...